run.py

```diff
@@ -986,13 +1039,31 @@ def engine_manifest():
         cancellable_engine = CancellableEngine(args)
 
     root_dir = args.voicevox_dir if args.voicevox_dir is not None else engine_root()
+
+    setting_file_path = args.setting_file
+
+    setting_loader = SettingLoader(setting_file_path)
+
+    settings = setting_loader.load_setting_file()
+
+    cors_policy_mode = (
+        args.cors_policy_mode
+        if args.cors_policy_mode is not CorsPolicyMode.localapps
+        else settings.cors_policy_mode
+    )
```

Comment:
なるほどーーーーーー。argparseでdefaultを`localapps`にしてるのでこのif判定が正しいですね！！
ただこうすると、argparse側のデフォルト値を変えたとき、こっち変え忘れてバグるのが想像できちゃいますね･･･ 😇 

これを避ける提案としては、
```py
parser = argparse.ArgumentParser(description="VOICEVOX のエンジンです。")
```
の上辺りに
```py
default_cors_policy_mode = CorsPolicyMode.localapps
```
として、`parser.add_argument`とここで`default_cors_policy_mode`を参照する手がありそうです。

修正ご面倒でしたら、そのままにして↓のFIXME付けるのでもいいかなと思いました！

```suggestion
    # FIXME: args.cors_policy_modeのデフォルト値が変わったらここの処理も変える
    cors_policy_mode = (
        args.cors_policy_mode
        if args.cors_policy_mode is not CorsPolicyMode.localapps
        else settings.cors_policy_mode
    )
```